lib/procedure.js

```diff
@@ -11,9 +11,10 @@ class Procedure {
     const exp = script(EMPTY_CONTEXT);
     if (typeof exp === 'object') {
       Object.assign(this, exp);
-      const { parameters, returns, concurrency } = exp;
+      const { parameters, returns, concurrency, validate } = exp;
```

Comment:
```suggestion
      const { parameters, returns, concurrency } = exp;
```
We don't need this after copying all fields from exp, see 1 line above.